src/Components/Organisms/ServiceCard/ServiceCard.tsx

```diff
@@ -0,0 +1,127 @@
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import AttachMoneyIcon from '@mui/icons-material/AttachMoney';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import CreateIcon from '@mui/icons-material/Create';
+import Typography from '@mui/material/Typography';
+import { Stack } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles({
+  btnsPosition: {
+    flexWrap: 'wrap',
+    alignItems: 'center',
+    justifyContent: 'center',
+    paddingBottom: 10,
+    '@media screen and (min-width: 600px)': {
+      justifyContent: 'flex-end',
+    },
+  },
+  btnStyles: {
+    borderRadius: 50,
+    minWidth: 200,
+    paddingTop: 9,
+    '@media screen and (min-width: 450px)': {
+      '&:first-child': {
+        marginRight: 8,
+      },
+    },
+  },
+  durationAndPrice: {
+    alignItems: 'center',
+    paddingBottom: 10,
+  },
+});
+
+const ServiceCard = () => {
+  const classes = useStyles();
+
+  const service = {
+    name: 'Masaż Gorącymi Kamieniami',
+    description:
+      'Odpręż się chłopie, to tutaj znajdziesz ukojenie. Zapomnij o troskach, zapomnij o brzemieniu.. Połóż się i nie myśl o niczym. Odpocznij,',
+    price: 150,
+    duration: 1.5,
+    imageUrl: require('../../../Assets/massage.jpg'),
```

Comment:
I'm not sure, but image probably will be stored as base64, it will be good to prepare functions to change format before displaying from base64 to image

Reply:
Thanks for the review. Done.